From 4a721ba2d1c0c70efe887f3da91a2152bef57e42 Mon Sep 17 00:00:00 2001
From: Daniel Steger <daniel.steger@xilinx.com>
Date: Fri, 30 Nov 2018 13:47:00 -0800
Subject: [PATCH 0499/1851] dt: bindings: media: xilinx: tpg: Add ppc dt
 bindings.

commit df92f9516cfeb837d4ba491b1d7f454e2621a0fe from
https://github.com/Xilinx/linux-xlnx.git

The tpg driver was modified to read the ppc from the devicetree.
This documentation update demonstrates how to set the ppc feature
for the tpg.

Signed-off-by: Daniel Steger <daniel.steger@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt
index d125a547abe8..f33282bd46a8 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt
@@ -23,6 +23,8 @@ Required properties:
 
 Optional properties:
 
+- xlnx,ppc: Pixels per clock. Valid values are 1, 2, 4 or 8.
+
 - xlnx,vtc: A phandle referencing the Video Timing Controller that generates
   video timings for the TPG test patterns.
 
@@ -47,6 +49,7 @@ Example:
 		reg = <0x40050000 0x10000>;
 		clocks = <&clkc 15>;
 
+		xlnx,ppc = <2>;
 		xlnx,vtc = <&vtc_3>;
 		timing-gpios = <&ps7_gpio_0 55 GPIO_ACTIVE_LOW>;
 
-- 
2.31.1

